src/customer_segmentation/SegmentAssigner.java

```diff
@@ -0,0 +1,27 @@
+package customer_segmentation;
+
+import java.math.BigDecimal;
+
+public class SegmentAssigner {
+
+    private static final BigDecimal VIP_INCOME = new BigDecimal("15000");
+    private static final BigDecimal VIP_ASSETS = new BigDecimal("1000000");
+    private static final BigDecimal PREMIUM_INCOME = new BigDecimal("7000");
+    private static final BigDecimal PREMIUM_ASSETS = new BigDecimal("200000");
+
+    public CustomerSegment assignSegment(Customer customer) {
```

Comment:
I don't think that we need to make `assignSegment` method public. Please use default access modifier.